frontend/pages/api/nextjs/cache/product-list.ts

```diff
@@ -0,0 +1,21 @@
+import { Duration } from '@lib/duration';
+import { ProductListFiltersInputSchema } from '@lib/strapi-sdk';
+import { withCache } from '@lib/swr-cache';
+import { findProductList } from '@models/product-list/server';
+import { ProductListSchema } from '@models/product-list/types';
+import { z } from 'zod';
+
+export default withCache({
+   endpoint: 'api/nextjs/cache/product-list',
+   statName: 'cache.findProductList',
+   variablesSchema: z.object({
+      filters: ProductListFiltersInputSchema(),
+      ifixitOrigin: z.string(),
+   }),
+   valueSchema: ProductListSchema.nullable(),
```

Comment:
Does that mean the model sometimes returns null too?

Reply:
`findProductList` can return null, if that's what you're asking. I might not be understanding the question though.

https://github.com/iFixit/react-commerce/blob/09d390a43ab91e18fd3961efd1b2351f3efc9ff7/frontend/models/product-list/server.ts#L38-L41